packages/muze-firebolt/src/side-effects/tooltip/strategies.js

```diff
@@ -63,9 +64,11 @@ const getRowContent = (fieldInf, context, dataInf, config) => {
     const { separator, classPrefix } = config;
     const dataLen = data.length;
     const values = [];
-    const index = fieldsConfig[field].index;
-    const interval = fieldsConfig[field].def.subtype === DimensionSubtype.TEMPORAL ? timeDiffs[field] : 0;
-    const formatterFn = getDefaultTooltipFormatterFn(formatters(val => val, interval, valueParser)[type],
+    const fieldObj = fieldsConfig[field];
+    const index = fieldObj.index;
+    const interval = fieldObj.def.subtype === DimensionSubtype.TEMPORAL ? timeDiffs[field] : 0;
+    const nf = fieldObj.def.type === FieldType.MEASURE ? fieldspace.fieldsObj()[field].numberFormat() : val => val;
```

Comment:
referencing `fieldObj` multiple times clutters the code.

Using destructuring is better. Plus `fieldObj.def` is referenced multiple times.